pkg/minikube/bootstrapper/bsutil/kubeadm.go

```diff
@@ -77,13 +78,25 @@ func GenerateKubeadmYAML(cc config.ClusterConfig, n config.Node, r cruntime.Mana
 		return nil, errors.Wrap(err, "cni")
 	}
 
-	podCIDR := cnm.CIDR()
-	overrideCIDR := k8s.ExtraOptions.Get("pod-network-cidr", Kubeadm)
-	if overrideCIDR != "" {
-		podCIDR = overrideCIDR
+	// Build podSubnet(s) based on IP family
+	family := strings.ToLower(k8s.IPFamily)
+	v4Pod := cnm.CIDR()
+	if o := k8s.ExtraOptions.Get("pod-network-cidr", Kubeadm); o != "" {
```

Comment:
Just curious, why can't we make either `cnm.CIDR()` take the ip family configuration and return a default that works for that family? That way, we can leave most of this untouched, right?

Am I missing something?